Sources/XiEditor/Search.swift

```diff
@@ -395,7 +395,7 @@ extension EditViewController {
             }
 
             if firstStatus.caseSensitive != nil {
-                query?.ignoreCase = !(statusQuery.caseSensitive != nil)
+                query?.caseSensitive = (statusQuery.caseSensitive != nil && statusQuery.caseSensitive!)
```

Comment:
I don't fully understand why is the `statusQuery.caseSensitive` negated here. Wasn't it just the case of the previous setting?

Reply:
Yeah, the wording `ignoreCase` and `caseSensitive` was source for confusion before. Case sensitive is basically the opposite of ignoring case (if casing is ignored then it's _not_ sensitive to casing), that's why it had to be negated before.